src/components/Footer/Info.tsx

```diff
@@ -1,13 +1,15 @@
-import React from "react";
+import React, { FC } from "react";
 
-export const Info = () => (
-  <div className="pt-20 pl-24 h-40">
-    <div className="text-10 font-lato font-black">
-      OZ CHAT - Centrum pre kreatívnu liečbu Arteterapiou
-    </div>
-    <div className="text-10 font-lato font-normal color-black">
-      <div>IČO: 42407681</div>
-      <div>Číslo účtu: VÚB SK13 0200 0000 0035 0237 4057</div>
-    </div>
+interface IProps {
+  title: string;
+  ico: number;
+  bankNumber: string;
```

Comment:
we don't need this props, this should be hardcoded we don't want to change this or reuse in different place